dev/how-to/deploy/orchestrated/tidb-in-kubernetes/aws-eks.md

```diff
@@ -233,31 +234,240 @@ By default, the terraform script will create a new VPC. You can use an existing
+  region = "us-west-1"
+}
+
+# create an ssh key to log in the bastion and the Kubernetes node
+module "key-pair" {
+  source = "../modules/aws/key-pair"
+
+  name = "another-eks-cluster"
+  path = "${path.cwd}/credentials/"
+}
+
+# provision a VPC
+module "vpc" {
+  source = "../modules/aws/vpc"
+
+  vpc_name = "another-eks-cluster"
+}
+
+# provision a EKS control plane with tidb-operator installed
+module "tidb-operator" {
+  source = "../modules/aws/tidb-operator"
+
+  eks_name           = "another-eks-cluster"
+  config_output_path = "credentials/"
+  subnets            = module.vpc.private_subnets
+  vpc_id             = module.vpc.vpc_id
+  ssh_key_name       = module.key-pair.key_name
+}
+
+# enforce helm to depend on the EKS
+resource "local_file" "kubeconfig" {
+  depends_on        = [module.tidb-operator.eks]
+  sensitive_content = module.tidb-operator.eks.kubeconfig
+  filename          = module.tidb-operator.eks.kubeconfig_filename
+}
+provider "helm" {
+  alias    = "eks"
+  insecure = true
+  install_tiller = false
+  kubernetes {
+    config_path = local_file.kubeconfig.filename
+  }
+}
+
+# provision a TiDB cluster in the EKS cluster
+module "tidb-cluster-a" {
+  source = "../modules/aws/tidb-cluster"
+  providers = { 
+    helm = "helm.eks"
+  }
+
+  cluster_name = "tidb-cluster-a"
+  eks          = module.tidb-operator.eks
+  ssh_key_name = module.key-pair.key_name
+  subnets      = module.vpc.private_subnets
+}
+
+# provision another TiDB cluster in the EKS cluster
+module "tidb-cluster-b" {
+  source = "../modules/aws/tidb-cluster"
+  providers = { 
+    helm = "helm.eks"
+  }
+  
+  cluster_name = "tidb-cluster-b"
+  eks          = module.tidb-operator.eks
+  ssh_key_name = module.key-pair.key_name
+  subnets      = module.vpc.private_subnets
+}
+
+# provision a bastion machine to access the TiDB service and worker nodes
+module "bastion" {
+  source = "../modules/aws/bastion"
+
+  bastion_name             = "another-eks-cluster-bastion"
+  key_name                 = module.key-pair.key_name
+  public_subnets           = module.vpc.public_subnets
+  vpc_id                   = module.vpc.vpc_id
+  target_security_group_id = module.tidb-operator.eks.worker_security_group_id
+  enable_ssh_to_workers    = true
+}
+
+# print the tidb hostname of tidb-cluster-a
+output "cluster-a_tidb-dns" {
+  description = "tidb service endpoints"
+  value       = module.tidb-cluster-a.tidb_hostname
+}
+
+# print the monitor hostname of tidb-cluster-b
+output "cluster-b_monitor-dns" {
+  description = "tidb service endpoint"
+  value       = module.tidb-cluster-b.monitor_hostname
+}
+
+output "bastion_ip" {
+  description = "Bastion IP address"
+  value       = module.bastion.bastion_ip
+}
+```
+
+As shown in the code above, you can omit most of the parameters in each of the module calls because there are reasonable defaults, and it is easy to customize the setup. For example, just delete the bastion module call if you do not need it.
```

Comment:
```suggestion
As shown in the code above, you can omit most of the parameters in each of the module calls because there are reasonable defaults, and it is easy to customize the configuration. For example, just delete the bastion module call if you do not need it.
```